static/scripts/audit-report/audit.ts

```diff
@@ -731,11 +628,8 @@ async function dbInit() {
 }
 
 async function resetInit() {
-  isComment = true;
-  commentPageNumber = 1;
-  issueList.splice(0, issueList.length);
+  permitList.splice(0, permitList.length);
   isGit = true;
```

Comment:
It is not clear to me what the purpose of this variable is.

Reply:
It holds the list of permits, the splice is to clear the list on reset I believe.

Reply:
No I mean `isGit = true;`